lib/utils/progress.go

```diff
@@ -311,22 +301,32 @@ func (p *progressPrinter) Stop() {
 	p.Lock()
 	defer p.Unlock()
 
-	if p.currentEntry != nil {
-		p.currentEntry.cancel()
-		if p.steps <= 0 {
-			diff := humanize.RelTime(p.start, time.Now(), "", "")
-			PrintStep(p.currentEntry.current, p.steps, fmt.Sprintf("%v finished in %v", p.title, diff))
-		} else if p.currentEntry.current == p.steps {
-			diff := humanize.RelTime(p.start, time.Now(), "", "")
-			PrintStep(p.currentEntry.current, p.steps, fmt.Sprintf("%v completed in %v", p.title, diff))
-		} else {
-			diff := humanize.RelTime(p.start, time.Now(), "", "")
-			PrintStep(p.currentEntry.current, p.steps, fmt.Sprintf("%v aborted after %v", p.title, diff))
-		}
+	if p.currentEntry == nil {
+		return
+	}
+	p.currentEntry.cancel()
+	if p.steps <= 0 {
+		diff := humanize.RelTime(p.start, time.Now(), "", "")
+		printStep(p.w, p.currentEntry.current, p.steps, fmt.Sprintf("%v finished in %v", p.title, diff))
+	} else if p.currentEntry.current == p.steps {
+		diff := humanize.RelTime(p.start, time.Now(), "", "")
+		printStep(p.w, p.currentEntry.current, p.steps, fmt.Sprintf("%v completed in %v", p.title, diff))
+	} else {
+		diff := humanize.RelTime(p.start, time.Now(), "", "")
+		printStep(p.w, p.currentEntry.current, p.steps, fmt.Sprintf("%v aborted after %v", p.title, diff))
 	}
 	p.currentEntry = nil
 }
 
+// printStep prints step instead of the progress bar
+func printStep(out io.Writer, current, target int, message string) {
+	if target > 0 {
+		fmt.Fprintf(out, "* [%v/%v] %v\n", current, target, message)
+	} else {
+		fmt.Fprintf(out, "%v\n", message)
```

Comment:
I stripped the time prefix here so progress notifications for other operations might be broken - I will fix this in the next PR.
I felt that time-prefixing should be done elsewhere (and it conflicts with the progress output in the installer).